_source/reference/streams.md

```diff
@@ -91,6 +91,28 @@ Inserts the content within the template tag after the element designated by the
 </turbo-stream>
 ```
 
+### Morph
+
+Replaces the element designated by the target dom id via morph.
+
+```html
+<turbo-stream action="morph" target="dom_id">
+  <template>
+    Content to replace the element.
+  </template>
+</turbo-stream>
+```
+
+The `children-only` attribute can be added to the `turbo-stream` element to morph only the children of the element designated by the target DOM id.
```

Comment:
Very minor but I'd go with *dom* instead of *DOM* for consistency with other references here.